services/mgmt/network/src/package_2019_09/models.rs

```diff
@@ -2089,7 +2089,7 @@ pub mod security_rule_properties_format {
         Icmp,
         Esp,
         #[serde(rename = "*")]
-        Asterisk,
+        U2a,
```

Comment:
I removed the special handling, so asterisk is `U2a`. https://www.unicodepedia.com/unicode/basic-latin/2a/asterisk/